### DISCUSSION TITLE
Description in README file Special_Config

### DISCUSSION THREAD

**Discussion by Firexstorm** [mriscoc/Ender3V2S1#759] [Ideas]
I'm currently working on several projects: e.g., building a 3d-printer, changing the motherboard in my E3V2,…
I therefore had the idea to add a small description of the options in the README file or in the python program in the special config folder.
It's easier to choose between the features IS, LA, MPC,... if you know what it is and does. At least for me (I'm still new to the topic)
Same with the thermistors and leveling. The rest should be self-explanatory.

**Comment by gsuresh2u** [comment]
Hi, to what motherboard you are changing ?
me upgraded Ender3 V2 to SKR MINI E3 V3 board witth ender extender kit

**Reply by Firexstorm** [reply]
Same upgrade: Ender 3 V2 and a SKR MINI E3 V3. 
My post is just an idea for folks who don't know everything about the settings and their function.

**Comment by mriscoc** [comment]
Check the Wiki: https://github.com/mriscoc/Ender3V2S1/wiki#special-versions

**Reply by Firexstorm** [reply]
I've read through the documentation.
I just thought about a small summary in the README file, nothing more.
But either way, thanks for the response!
